cloudahoywriter.lua

```diff
@@ -6,29 +6,59 @@
 -- Code conventions
 --  - Use local when possible
 --  - Globals are prefixed with CAWR_
+--  - Method names use_underscores
+--  - Variable names useCamelCase
+--  - Constant names are CAPITAL_WITH_UNDERSCORE
 -----------------------------------------------
-local versionNum = '0.0.1'
+local versionNum = '0.0.2'
 
 require('graphics')
 
 -------------------- CONSTANTS --------------------
 local LOG_INTERVAL_SECS = 0.3
 local SECONDS_PER_MINUTE = 60
 local SECONDS_PER_HOUR = SECONDS_PER_MINUTE * 60
+local NOTIFICATION_SECS = 10
+-- Start or stop recording when ground speed crosses this speed.
+local AUTO_RECORDING_GROUND_SPEED_MPS = 3 -- Dataref is in m/s
+-- Stop recording when the ground speed has been below the speed above
+-- for more thsn this time.
+local AUTO_STOP_TIME_SECS = 10 * SECONDS_PER_MINUTE
+-- Disable automatic recording state change within this many seconds of the
+-- making a state change.
+local AUTO_RECORDING_DISABLE_SECS = 5 * SECONDS_PER_MINUTE
 local FLIGHTDATA_DIRECTORY_NAME = 'flightdata'
 local OUTPUT_PATH_NAME =  SYSTEM_DIRECTORY .. 'Output/' .. FLIGHTDATA_DIRECTORY_NAME
 
 -------------------- STATE --------------------
 local lastWriteTime = nil
 local recordingStartOsTime = nil
 local recordingStartSimTime = nil
-local recordingLuaRun = nil
 local recordingDisplayTime = '0:00:00'
+local recordingNotifyStateChangeTimeEnd = nil
```

Comment:
Could you add a comment explaining this?

Reply:
Done. Also renamed variable to `forceShowUiTimeEnd`